custom-vars.el

```diff
@@ -0,0 +1,39 @@
+(custom-set-variables
+ ;; custom-set-variables was added by Custom.
+ ;; If you edit it by hand, you could mess it up, so be careful.
+ ;; Your init file should contain only one such instance.
+ ;; If there is more than one, they won't work right.
+ '(doom-modeline-bar-width 6)
+ '(doom-modeline-buffer-file-name-style 'truncate-except-project)
+ '(doom-modeline-height 15)
+ '(doom-modeline-minor-modes t)
+ '(evil-respect-visual-line-mode t)
+ '(evil-undo-system 'undo-tree)
+ '(evil-want-C-i-jump nil)
+ '(evil-want-C-u-scroll t)
+ '(evil-want-integration t)
+ '(evil-want-keybinding nil)
+ '(fast-but-imprecise-scrolling t)
+ '(global-auto-revert-non-file-buffers t)
+ '(kill-do-not-save-duplicates t)
+ '(large-file-warning-threshold 100000000)
+ '(safe-local-variable-values
+   '((indent-tabs-mode . allways)
+     (whitespace-mode . t)
+     (firestarter . "./build.sh;notify-send build done! ")
+     (electric-indent-mode . t)
+     (eval
+      (setq ecc-run-function 'fennel-run-function))
+     (org-support-shift-select . t)
+     (tab-always-indent . t)
+     (lua-indent-level . 2)))
+ '(scroll-conservatively 101)
+ '(scroll-margin 0)
+ '(scroll-preserve-screen-position t)
+ '(visible-bell 1))
+(custom-set-faces
+ ;; custom-set-faces was added by Custom.
+ ;; If you edit it by hand, you could mess it up, so be careful.
+ ;; Your init file should contain only one such instance.
+ ;; If there is more than one, they won't work right.
+ )
```

Comment:
This whole file seems like an accidental commit. Was this really intended? Otherwise please rebase to remove the file.

Reply:
Yeah that was accidental

Reply:
You will need to remove the file from your PR as it does not belong.